File: backend/config/dbconnect.py
import mysql.connector

# Database connection details
host = 'psmedical'  # Replace with your host (e.g., localhost or an IP address)
database = 'dbmedical'  # Replace with your database name
user = 'root'  # Replace with your MySQL username
password = 'healty'  # Replace with your MySQL password

try:
    # Establish a connection to the database
    connection = mysql.connector.connect(
        host=host,
        database=database,
        user=user,
        password=password
    )
    
    if connection.is_connected():
        print(f"Successfully connected to the {database} database.")
        
        # Creating a cursor object using the connection
        cursor = connection.cursor()
        
        # Example query: Selecting data from the users table
        cursor.execute("SELECT * FROM users")
        
        # Fetch all the rows from the executed query
        result = cursor.fetchall()
        
        # Loop through the result and print each row
        for row in result:
            print(row)
        
        # Close the cursor and connection
        cursor.close()

except mysql.connector.Error as err:
    print(f"Error: {err}")
    
finally:
    # Close the connection if it's open
    if connection.is_connected():
        connection.close()
        print("MySQL connection is closed.")

File: backend/config/dbconnect.php
<?php
$host = 'psmedical.com'; // Database host
$user = 'pacha';      // Database username
$pass = '';          // Database password
$dbname = 'dbmedical'; // Database name

// Create connection
$deal = new mysqli($host, $user, $pass, $dbname);

// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $deal->connect_error);
}
?>
